plugins/content/transcript/lib/TranscriptAssetType.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * @package plugins.transcript
+ * @subpackage lib.enum
+ */
+class TranscriptAssetType extends AttachmentAssetType
```

Comment:
Implement IKalturaPluginEnum and assetType instead of extending AttachmentAssetType, it will be more clear that it's dynamic enum expansion and not class extension.